Makefile

```diff
@@ -1,40 +1,47 @@
 all: build
+
 export EMACS ?= emacs
+
 EMACSFLAGS = -L .
-CASK = cask
+
+EASK = eask
+
 VERSION = $(shell git describe --tags --abbrev=0 | sed 's/^v//')
+
 PKG = fsharp-mode
 
-elpa-$(EMACS):
-	$(CASK) install
-	$(CASK) update
-	touch $@
+ci: build compile test
```

Comment:
Here is the CI entry, now we only have three tasks.

* `build` (test `packaging` and `installing`)
* `compile` - byte-compile source files (files are defined in `Eask`-file)
* test (left unchanged)